proposals/0317-sdl-protocol-security-specification.md

```diff
@@ -137,108 +152,13 @@ The TLS Payload is sent in a binary header. The size is 12 bytes which matches t
   </tr>
 </table>
 
-#### 4.7.1.3 Hybrid Query Payload
-
-The security query is able to contain JSON data as well as binary data. During the handshake the TLS handshake data is sent as binary data. In case of an error, a notification is sent with an error code and error message as JSON data. See "Error handling" section for details.
-
-<table width="100%">
-  <tr><td align="center">Binary Query Header</td></tr>
-  <tr><td align="center">JSON Data</td></tr>
-  <tr><td align="center">Binary Data</td></tr>
-</table>
-
-### 4.7.2 Start Service 
```

Comment:
Per Item 8 in [this comment](https://github.com/smartdevicelink/sdl_evolution/issues/1070#issuecomment-724282962) outlining Steering Committee requested revisions, the following information should be added to this section:
>In 4.7.2 (Start Service), describe RPC message protection capabilities to lift existing service sessions to become protected. Also RPC service must start unprotected.

I noticed some of this information is included in Section 7 (Secured Communication), so you could also include a reference to Section 7 here, but this section (4.7.2) should at least mention sending a second start service with the encrypted bit set in order to elevate a service into an encryption service.

Reply:
@theresalech section 4.7.2 has been removed according to new requested order in the comment you mentioned. By that reason item 8 should be described in the Section 7 per my understanding. I updated some paragrapghs in Section 7 to mention sending of a second StartService and how this elevation works for another type of services. Please check for my updates in https://github.com/smartdevicelink/sdl_evolution/pull/1138/commits/2a6995a428cb531a3f50bfd8253e96dcffc3548f and let me know if that information is enough

Reply:
@AKalinich-Luxoft , thank you for adding to Section 7! Although 4.7.2 was removed, there should still be details about "lifting" a service into an encrypted service in the existing Section 4. (Establishing Communication). Can you please include in the proposal that this will be added as a new section: 4.2.5?

Reply:
@theresalech sure. Added section 4.2.5 and I moved one relevant paragraph from section 7 to 4.2.5 and also refer to section 7 from there. See https://github.com/smartdevicelink/sdl_evolution/pull/1138/commits/717d6bf8ca90b75bfd9ecfac2131ec568d9c3189